Framework/DataHandling/src/LoadEventNexus.cpp

```diff
@@ -1982,10 +1116,16 @@ void LoadEventNexus::loadEvents(API::Progress *const prog,
 
   for (size_t i = bank0; i < bankn; i++) {
     // We make tasks for loading
-    if (bankNumEvents[i] > 0)
-      pool.schedule(new LoadBankFromDiskTask(
+    if (bankNumEvents[i] > 0) {
+      LoadBankFromDiskTask *task = new LoadBankFromDiskTask(
```

Comment:
What is the reason we are removing the parallelization? Is there evidence it does help with performance?